src/ZfSnapPhpDebugBar/Service/PhpDebugBarFactory.php

```diff
@@ -38,7 +38,7 @@ public function createService(Locator $serviceLocator)
         $debugbar->addCollector(new ConfigCollector($appConfig, 'ApplicationConfig'));
 
         // Db profiler
-        if ($serviceLocator->has($dbServiceName) && isset($config['db']['driver'])) {
```

Comment:
Ok, but we try to create a db adapter which throw exception in some cases (I don't remember which one - missing test for that :-( )